schemas/campaigns/experience.schema.json

```diff
@@ -38,6 +38,14 @@
           "type": "string",
           "format": "date-time",
           "description": "The date when the activity expires. Different publishing channels might use the end date in implementation-dependent ways."
+        },
+        "xdm:thumbnail": {
+          "title": "Thumbnail",
+          "type": "string",
+          "description": "Thumbnail image representing the experience"
+        },
+        "xdm:parent": {
+          "$ref": "https://ns.adobe.com/xdm/campaigns/experience"
```

Comment:
Dmitry has spent quite a bit of time thinking about how to model relationships. He's still collecting arch feedback, but his current (I believe) proposal is here: https://wiki.corp.adobe.com/display/DMSArchitecture/XDM+Relationships+in+XCORE. This scheme of using a reference object to model relationships seems like it should work well.

Reply:
Yes, I think so. Our hierarchy, when mapped to Engage, should eventually look like this:
* activities may have no parent or be the children of experiences
* an experience can be a children of another experience